src/scenes/end.js

```diff
@@ -7,19 +7,17 @@ export default class EndScene extends Phaser.Scene {
 
     preload() {}
 
-    create() {
+    create(data) {
         elements.appEnd.style.visibility = "visible";
         elements.appEnd.style.display = "flex";
         elements.appEnd.style.marginTop = "10%";
-
-        // TODO: Update display score once gameState is implemented
-        // elements.endScoreDisplay.innerHTML = `Score: ${}`
+        elements.endScoreDisplay.innerHTML = `Score: ${Math.trunc(data.score)}`;
```

Comment:
Why the `Math.trunc`? I feel like you could remove it, right?
```suggestion
        elements.endScoreDisplay.innerHTML = `Score: ${data.score}`;
```

Reply:
This happens when I remove Math.trunc(data.score).

<img width="1570" alt="Screen Shot 2019-06-28 at 10 01 05 AM" src="https://user-images.githubusercontent.com/26859409/60347712-fddaf580-998b-11e9-8f09-54b4ea5c1c49.png">

Should keep it?

Reply:
LOL. Why is that happening? Can you find where the score is becoming this odd thing?

Reply:
That's interesting, I thought there that `score` was being `Math.floor()`ed somewhere in **game.js**

Reply:
I'll try to find out